server/events.go

```diff
@@ -575,12 +575,9 @@ func (s *Server) initEventTracking() {
 	}
 	s.sys.inboxPre = subject
 	// This is for remote updates for connection accounting.
-
-	for _, subj := range []string{accConnsEventSubjOld, accConnsEventSubjNew} {
-		subject = fmt.Sprintf(subj, "*")
-		if _, err := s.sysSubscribe(subject, s.remoteConnsUpdate); err != nil {
-			s.Errorf("Error setting up internal tracking for %s: %v", subject, err)
-		}
+	subject = fmt.Sprintf(accConnsEventSubjOld, "*")
```

Comment:
But why introduce publish on `accConnsEventSubjNew` subject if we never subscribe on that subject. What is the plan then? Replace subscription on "old" with "new" at one point?
Not sure about all that..

Reply:
Or is the new subject subscription for apps to listen to, not the server?

Reply:
ok, got it.